examples/plot_sleep_staging_usleep.py

```diff
@@ -0,0 +1,355 @@
+"""
+Sleep staging on the Sleep Physionet dataset using U-Sleep network
+==================================================================
+
+.. note::
+    Please take a look at the simpler sleep staging example
+    :ref:`here <sphx_glr_auto_examples_plot_sleep_staging.py>`
+    before going through this example. The current example uses a more complex
+    architecture and a sequence-to-sequence (seq2seq) approach.
+
+This tutorial shows how to train and test a sleep staging neural network with
+Braindecode. We adapt the U-Sleep approach of [1]_ to learn on sequences of EEG
+windows using the openly accessible Sleep Physionet dataset [2]_ [3]_.
+
+.. warning::
+    The example is written to have a very short excecution time.
+    This number of epochs is here too small and very few recordings are used.
+    To obtain competitive results you need to use more data and more epochs.
```

Comment:
Given what you observe can you be more specific? How many epochs will lead to
what accuracy (roughly). Something like : Taking for example at least 30 recordings
to training and doing 40 epochs should lead to a balanced accuracy close to 0.8.

Reply:
can you also put the same warning in plot_sleep_staging.py  ?

Reply:
Yes and also add the changing of the depth parameter, in case that was for execution time @Tgnassou